homework/kaydunov_denis/shopping_list_application_part2/src/test/java/github/kaydunovdenis/bean/ProductTest.java

```diff
@@ -0,0 +1,139 @@
+package github.kaydunovdenis.bean;
+
+import nl.jqno.equalsverifier.EqualsVerifier;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.math.BigDecimal;
+
+public class ProductTest {
+    private Product product;
+
+    @Before
+    public void setUp() {
+        product = getDefaultProduct();
```

Comment:
as you have fabric, no need to instantiate product every time here, no need to have product as a field in the test.

Reply:
has been fixed